pkgs/servers/search/elasticsearch/plugins.nix

```diff
@@ -94,10 +94,10 @@ in {
   elasticsearch_kopf = esPlugin rec {
     name = "elasticsearch-kopf-${version}";
     pluginName = "elasticsearch-kopf";
-    version = "1.5.7";
+    version = "2.1.1";
```

Comment:
As elasticsearch 1.7.x is EOL (as of 2017-01-16, before the release of 17.03) I do not think a backport is needed.